.github/workflows/large-pr-labeler.yml

```
@@ -2,8 +2,12 @@ name: Label Large PR
 
 on:
   pull_request:
+permissions:
     types: [opened, synchronize, reopened]
 
+permissions:
```

actionlint (1.7.12)

[error] key "permissions" is duplicated in "workflow" section. previously defined at line:5,col:1

(syntax-check)